OptimizelySDK/Odp/OdpConfig.cs

```diff
@@ -81,27 +81,6 @@ public OdpConfig(string apiKey, string apiHost, List<string> segmentsToCheck)
             SegmentsToCheck = segmentsToCheck ?? new List<string>(0);
         }
 
-        /// <summary>
```

Comment:
If the entire odpconfig is replaced for update, then we do not need t"volitile" for internal fields.